modules/models/src/main/java/org/eclipse/xpanse/modules/models/view/UserDeployableServiceVo.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+
+package org.eclipse.xpanse.modules.models.view;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import org.eclipse.xpanse.modules.models.enums.Category;
+import org.eclipse.xpanse.modules.models.enums.Csp;
+import org.eclipse.xpanse.modules.models.enums.ServiceState;
+import org.eclipse.xpanse.modules.models.resource.Billing;
+import org.eclipse.xpanse.modules.models.resource.DeployVariable;
+import org.eclipse.xpanse.modules.models.resource.Flavor;
+import org.eclipse.xpanse.modules.models.resource.Region;
+import org.springframework.hateoas.RepresentationModel;
+
+/**
+ * Defines view object for end-user query registered service.
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class UserDeployableServiceVo extends RepresentationModel<UserDeployableServiceVo> {
+
+    @NotNull
+    @Schema(description = "ID of the registered service.")
+    private UUID id;
+
+    @NotNull
+    @Schema(description = "The catalog of the service")
+    private Category category;
+
+    @NotNull
+    @NotBlank
+    @Schema(description = "The name of the managed service")
+    private String name;
+
+    @NotNull
+    @NotBlank
+    @Schema(description = "The version of the managed service")
+    private String version;
+
+    @NotNull
+    @Schema(description = "The Cloud Service Provider.")
+    private Csp csp;
+
+    @NotEmpty
+    @Schema(description = "The regions of the Cloud Service Provider")
+    private List<Region> regions;
+
+    @NotNull
+    @NotBlank
+    @Schema(description = "The description of the managed service")
+    private String description;
+
+    @NotNull
+    @NotBlank
+    @Schema(description = "The namespace of the managed service")
+    private String namespace;
+
+    @NotNull
+    @NotBlank
+    @Schema(description = "The icon of the managed service")
+    private String icon;
+
+    @NotNull
+    @Schema(description = "The variables for the deployment, which will be passed to the deployer")
+    private List<DeployVariable> variables;
+
+    @NotNull
+    @Schema(description = "The flavors of the managed service")
+    private List<Flavor> flavors;
+
+    @NotNull
+    @Schema(description = "The billing policy of the managed service")
+    private Billing billing;
+
+    @NotNull
+    @Schema(description = "Time of register service.")
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    private Date createTime;
+
+    @NotNull
+    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
+    @Schema(description = "Time of update service.")
+    private Date lastModifiedTime;
```

Comment:
```suggestion
    private Date updateTime;
```
How about updateTime?

Reply:
I think lastModifyTime is more clear.

Reply:
As we just have one copy, the prefix `last` is not needed here.

Reply:
This field is also in DB entity. We can change it  in a new PR.